R/pkg/NAMESPACE

```diff
@@ -236,6 +236,7 @@ exportMethods("%<=>%",
               "current_date",
               "current_timestamp",
               "hash",
+              "data_type",
```

Comment:
I know this list isn't completely sorted, let's sort this?
you can move "hash" and "data_type"